README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion - **The use of a method/application inside itself**
+- Recursive Case - **A problem that is divided into smaller problems**
```

Comment:
This is not correct.

A recursive case is the case in which the method actually calls itself.